config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-    jitsiMeetURL: "https://meet.jit.si/"
+    jitsiMeetURL: "https://meet.jit.si/",
+    handleAuthorization: false
```

Comment:
Question: why would someone not want authorization to be handled? Also, why is it false by default?